tensorflow/lite/micro/examples/hello_world/evaluate_test.cc

```diff
@@ -24,6 +24,20 @@ limitations under the License.
 #include "tensorflow/lite/micro/system_setup.h"
 #include "tensorflow/lite/schema/schema_generated.h"
 
+namespace {
+using HelloWorldOpResolver = tflite::MicroMutableOpResolver<1>;
+alignas(alignof(HelloWorldOpResolver)) uint8_t
+    op_resolver_buffer[sizeof(HelloWorldOpResolver)];
+}  // namespace
+
+namespace tflite {
```

Comment:
Delete. The OpResolver function can go in the anon namespace.

Reply:
done